### PR TITLE
[cli] Create cvd with local image.

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: pkg/cli/cli.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ func buildServiceBuilder(builder client.ServiceBuilder) serviceBuilder {
 			ErrOut:         c.ErrOrStderr(),
 			RetryAttempts:  3,
 			RetryDelay:     5 * time.Second,
-			ChunkSizeBytes: 10 * 1024 * 1024,
+			ChunkSizeBytes: 16 * 1024 * 1024,
```
Comment (jemoreira) [inline pkg/cli/cli.go:147]: Please extract this change in constant value, along with the one in client.go to a separate commit and use the "rebase and merge" strategy instead of "squash and merge"

**Review by jemoreira — APPROVED** [review]

File: pkg/cli/cli.go (hunk 1 above)
Comment (ser-io) [reply pkg/cli/cli.go]: done, in client.go we already use a constant variable.
